### PR TITLE
Functionize token verification

### PR DESCRIPTION
**Pull request by themarkrizkallah**
Please review my implementation of verify_token() as I currently have no way of testing it myself @ImShady

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -68,6 +61,14 @@ def button_click():
 
     return jsonify(result)
 
+def verify_token(token):
```
Comment (ImShady) [inline main.py]: This function shouldn't belong in main, maybe put it in Tubey and use `tubey.verify_token(<token>)`?

File: lib/slack.py [hunk 2]
```diff
@@ -120,6 +121,13 @@ def search(self, search_query):
 
         return results['videos']
 
+    def verify_token(token):
```
Comment (ImShady) [inline lib/slack.py]: This won't compile, all functions inside a class must have `self` as the first argument

**Comment by themarkrizkallah** [comment]
I moved the function to slack.py. Please review my changes @ImShady

File: main.py [hunk 3]
```diff
@@ -26,18 +26,16 @@
 def slash_command():
     # Parse the command parameters, validate them, and respond
     token = request.form.get('token', None)
-    verif_token = Config.get_variable('tubey', 'verif_token')
     print(request.form)
 
-    # Validate the request parameters
-    if token != verif_token:
-        abort(401)  # Unauthorized request. If you're not Slack, go away
+    tubey = Tubey()
```
Comment (ImShady) [inline main.py]: Line not needed, we already have a global Tubey object declared on line 23

File: main.py [hunk 4]
```diff
@@ -47,10 +45,9 @@ def button_click():
     payload = json.loads(request.form.get('payload', None))
     print(payload)
 
-    verif_token = Config.get_variable('tubey', 'verif_token')
+    tubey = Tubey()
```
Comment (ImShady) [inline main.py]: Same here

File: main.py [hunk 5]
```diff
@@ -68,6 +65,7 @@ def button_click():
 
     return jsonify(result)
 
+
```
Comment (ImShady) [inline main.py]: too many linez

**Comment by themarkrizkallah** [comment]
I made the changes you suggested @ImShady

**Review by ImShady — APPROVED** [review]

**Pull request merged**